src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -73,7 +73,7 @@ public Limelight(String name) {
     limelightHasTarget = Shuffleboard.getTab("Driver").add(name + "has target", false).getEntry();
   }
 
-  /* now its time for getter method chaingun, which I have to write manually because VS Code */
+  /* now its time for getter method chain gun, which I have to write manually because VS Code */
```

Comment:
you put a hyphen in the last one. Be consistent.

Reply:
ugh